crates/nu-protocol/src/debugger/profiler.rs

```diff
@@ -277,6 +299,16 @@ fn collect_data(
         "parent_id" => Value::int(parent_id.0 as i64, profiler_span),
     };
 
+    if profiler.collect_lines {
+        if let Some((fname, line_num)) = find_file_of_span(engine_state, element.element_span) {
+            row.push("file", Value::string(fname, profiler_span));
+            row.push("line_num", Value::int(line_num as i64, profiler_span));
+        } else {
+            row.push("file", Value::nothing(profiler_span));
+            row.push("line_num", Value::nothing(profiler_span));
```

Comment:
One last thing, can we rename this column to just `line`?
```suggestion
            row.push("line", Value::nothing(profiler_span));
```

Reply:
I made the suggestion wrong, there's also another `line_num` above.